### PR TITLE
Initialize content action.

### PR DESCRIPTION
**Pull request by samueleresca**
Related to https://github.com/apache/incubator-pekko-site/issues/56

### CODE REVIEW

**Comment by samueleresca** [comment]
I'm investigating why the GH action is not triggered

**Comment by pjfanning** [comment]
I suspect that CI Actions is not enabled in this repo. Maybe we'll need to raise an INFRA JIRA to get it looked at. We can't access the GitHub Setup tab to enable it for themselves.

**Comment by samueleresca** [comment]
Requirements to enable GH Actions on the repo: https://github.com/apache/incubator-pekko-site/pull/62. See comment in the infra ticket

File: .github/workflows/site.yml [hunk 1]
```diff
@@ -1 +1,33 @@
...
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: 0
+      - name: Setup Java 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 8
```
Comment (pjfanning) [inline .github/workflows/site.yml:20]: we need java 11

File: .github/workflows/site.yml [hunk 2]
```diff
@@ -1 +1,33 @@
...
+    name: Check content folder
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: 0
+      - name: Setup Java 8
+        uses: actions/setup-java@v3
```
Comment (pjfanning) [inline .github/workflows/site.yml:17]: could you use v4?

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**